pkg/bee/client.go

```diff
@@ -750,6 +767,54 @@ func (c *Client) UploadFile(ctx context.Context, f *File, o api.UploadOptions) (
 	return
 }
 
+func (c *Client) UploadActFile(ctx context.Context, f *File, o api.UploadOptions) (err error) {
+	h := fileHasher()
+	r, err := c.api.Act.Upload(ctx, f.Name(), io.TeeReader(f.DataReader(), h), o)
+	if err != nil {
+		return fmt.Errorf("upload ACT file: %w", err)
+	}
+
+	f.SetAddress(r.Reference)
+	f.SetHistroryAddress(r.HistoryAddress)
+	f.SetHash(h.Sum(nil))
+
+	return nil
+}
+
+func (c *Client) AddActGrantees(ctx context.Context, f *File, o api.UploadOptions) (err error) {
+	h := fileHasher()
+	r, err := c.api.Act.AddGrantees(ctx, f.Name(), io.TeeReader(f.DataReader(), h), o)
```

Comment:
should we have also some `defer r.Close()` ? Also in below similar situations?

Reply:
r is ActGranteesResponse struct